### ISSUE TITLE
Error in `grid.newpage()`: could not open file

### ISSUE THREAD

**Issue by pdiakumis** [umccr/RNAsum#147]
Using RNAsum v0.6.2 we get the following:

```
2024-03-27T05:15:37.0887344Z stderr F
2024-03-27T05:15:37.088778618Z stderr F
2024-03-27T05:15:37.088784662Z stderr F processing file: rnasum.Rmd
2024-03-27T05:15:37.23150511Z stdout F 1/215
2024-03-27T05:15:37.232707626Z stdout F 2/215 [prep_knitr_hooks]
2024-03-27T05:15:37.248599514Z stdout F 3/215
2024-03-27T05:15:37.248709967Z stdout F 4/215 [unnamed-chunk-1]
2024-03-27T05:15:47.317253074Z stdout F 5/215
2024-03-27T05:15:47.317372926Z stdout F 6/215 [read_data]
2024-03-27T05:17:11.41313952Z stdout F 7/215
2024-03-27T05:17:11.413184086Z stdout F 8/215 [chunk_eval]
2024-03-27T05:17:11.428511318Z stdout F 9/215
2024-03-27T05:17:11.428599791Z stdout F 10/215 [dragen_mapping_metrics]
2024-03-27T05:17:11.437759931Z stdout F 11/215
2024-03-27T05:17:11.437848956Z stdout F 12/215 [mysql_populate]
2024-03-27T05:17:11.446202984Z stdout F 13/215
2024-03-27T05:17:11.446302922Z stdout F 14/215 [cancer_genes_prep]
2024-03-27T05:17:14.829894097Z stdout F 15/215
2024-03-27T05:17:14.829933553Z stdout F 16/215 [goi_summary]
2024-03-27T05:17:14.872728788Z stdout F 17/215
2024-03-27T05:17:14.872778221Z stdout F 18/215 [goi_annotation]
2024-03-27T05:17:14.910769099Z stdout F 19/215
2024-03-27T05:17:14.910853064Z stdout F 20/215 [library_size_plot]
2024-03-27T05:17:15.112356279Z stderr F
2024-03-27T05:17:15.112381052Z stderr F Quitting from lines 364-390 [library_size_plot] (rnasum.Rmd)
2024-03-27T05:17:15.12567641Z stderr F Error in `grid.newpage()`:
2024-03-27T05:17:15.125696933Z stderr F ! could not open file '/omdcjk/SBJ03034__L2300038/MDX220434.results/InputDataPlots/library_size.png'
2024-03-27T05:17:15.125704513Z stderr F Backtrace:
2024-03-27T05:17:15.12571015Z stderr F  1. ggplot2::ggsave(...)
2024-03-27T05:17:15.125714175Z stderr F  3. ggplot2:::grid.draw.ggplot(plot)
2024-03-27T05:17:15.125717147Z stderr F  5. ggplot2:::print.ggplot(x)
2024-03-27T05:17:15.125720358Z stderr F  6. grid::grid.newpage()
2024-03-27T05:17:15.127586613Z stderr F Execution halted
```

Think it has something to do with dropping gpgr which in turn dropped some of the visualisation/graphics dependencies (e.g. ragg, textshaping - see https://github.com/umccr/RNAsum/commit/5d5df0361daf41cb7341ff88de8054560a3ee527).
This shouldn't really be happening. Slack at <https://umccr.slack.com/archives/C04F5SMUH8C/p1711522864045829>.

**Comment by pdiakumis** [comment]
Can't reproduce on Gadi or a local Docker instance, added back gpgr to see if it sorts itself out.

**Comment by pdiakumis** [comment]
Fixed by #147.
